src/Domain/CoordinateMaps/Affine.cpp

```diff
@@ -93,9 +96,19 @@ bool operator==(const CoordinateMaps::Affine& lhs,
   template tnsr::Ij<tt::remove_cvref_wrap_t<DTYPE(data)>, 1, Frame::NoFrame> \
   Affine::inv_jacobian(const std::array<DTYPE(data), 1>& source_coords) const;
 
-GENERATE_INSTANTIATIONS(INSTANTIATE, (double, DataVector,
-                                      std::reference_wrapper<const double>,
-                                      std::reference_wrapper<const DataVector>))
+GENERATE_INSTANTIATIONS(
+    INSTANTIATE, (double, DataVector,
```

Comment:
Can you do something like `INSTANTIATE, BOOST_PP_TUPLE_CAT((...), AUTODIFF_MAP_INSTANTIATIONS)`? Then you only have to define the map types in one place. We might actually want to do the some for the non-autodiff types, so we'd have `...TUPLE_CAT(MAP_TYPES, MAP_AUTODIFF_TYPES)`